app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("Friends should buy masks")
```

Comment:
fix message, now it looks like to visit a cafe everyone should buy a mask, but they should just wear them

Reply:
fixed